src/main/scala/dotty/dokka/tasty/TypesSupport.scala

```diff
@@ -32,14 +31,22 @@ trait TypesSupport:
         case List(single) => single
         case other => other.reduce((r, e) => r ++ texts(", ") ++ e)
     
+    private def isRepeated(tpeAnnotation: Term) =
```

Comment:
@AleksanderBG Any idea why this does not work? Maybe there is a better way to detect repeated parameters (varags)?

I based my logic on `RefinedPrinter` and it seems to work fine

Reply:
May this bug be caused by serializaition to TASTY?

Reply:
It seems like comparing to `defn.RepeatedParamClass` is the only way to check if something is a repeated param. I'll open a bug in Dotty and let you know when it's fixed.